src/opencl_bitlocker_fmt_plug.c

```diff
@@ -128,10 +128,13 @@ static struct fmt_main *self;
 #define SEED			1024
 
 static const char *warn[] = {
-        "xfer: ",  ", init: " , ", crypt: ", ", res xfer: "
+        "xfer: ",  ", ",  ", ",  ", ",  ", ",
+        ", ",  ", ",  ", ",  ", ",  ", ",
+        ", ",  ", ",  ", prepare: " ,  ", xfer: ",  ", crypt: ",
+        ", final: ",  ", res xfer: "
 };
 
-static int split_events[] = { 2, -1, -1 };
+static int split_events[] = { 14, 13, -1 };
```

Comment:
Well, I'm using it to compute the average and show the variation (see crypt and var):
```
xfer: 8.264ms, crypt: 119x108.091ms, xfer back: 14.997ms, prep: 192.114ms, pp: 6.325ms, final: 5.904ms, var: 108.144ms/108.064ms
```
So, I don't want the 16x. 
* That said, I can think about something like: `crypt: 16x(71.767ms + 4.064us)`
*****

* The kernel itself can control *currentIterPtr*, but the no gain is expected.
* BTW: It is not 16x, it is 4096x.